doc/src/manual/integers-and-floating-point-numbers.md

```diff
@@ -178,7 +178,7 @@ of the binary data item is the minimal needed size, if the leading digit of the
 `0`. In the case of leading zeros, the size is determined by the minimal needed size for a
 literal, which has the same length but leading digit `1`. That allows the user to control
 the size.
-Values, which cannot be stored in `UInt128` cannot be written as such literals.
+Values which cannot be stored in `UInt128` cannot be written as such literals.
 
```

Comment:
I am not a native English speaker, but shouldn't a subclause that is essential (which is why the comma is not at its place), use "that" instead of "which", i.e. either of the following would be grammatically correct, but the latter does of course not have the correct meaning:

Values that cannot be stored in `UInt128` cannot be written as such literals.

Values, which cannot be stored in `UInt128`, cannot be written as such literals.

Reply:
Lose the double negative:
"Only values that can be stored in `UInt128` can be written as such literals."